### PR TITLE
Update ome-faq.yml

### PR DESCRIPTION
**Pull request by msbemba**
Tested and able to view encrypted email and attachments on outlook for IOS and android

### CODE REVIEW

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/5037

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/compliance/ome-faq.yml [hunk 1]
```diff
@@ -215,7 +215,7 @@ sections:
           
           - Users can open protected mails in a shared mailbox where the shared mailbox received a protected mail as part of a distribution group.
           
-          - Users can view attachments that inherit protection from email when they use Outlook for Windows, Outlook for Mac, and Outlook on the web.
+          - Users can view attachments that inherit protection from email when they use Outlook for Windows, Outlook for Mac, Outlook for Android, Outlook for iOS and Outlook on the web.
```
Comment (JohanFreelancer9) [inline microsoft-365/compliance/ome-faq.yml]:
```suggestion
          - Users can view attachments that inherit protection from email when they use Outlook for Windows, Outlook for Mac, Outlook for Android, Outlook for iOS, and Outlook on the web.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review. Minor edit.

**Comment by yogkumgit** [comment]
@msbemba

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@KCCross Please merge this PR. Thanks

**Comment by yogkumgit** [comment]
@KCCross Please merge this Pr. Thanks

**Comment by yogkumgit** [comment]
@KCCross Gentle reminder.

**Comment by KCCross** [comment]
Samson, please review. Difference between functional and supported.

**Comment by yogkumgit** [comment]
@samschan-ms Gentle reminder

**Comment by samschan-ms** [comment]
@KCCross please add a note that android and iOS would use the Office mobile app to view the encrypted attachments, and not directly in Outlook mobile.

**Comment by msbemba** [comment]
@KCCross  made changes per samschan-ms advise

**Review by samschan-ms — APPROVED** [review]

**Comment by yogkumgit** [comment]
@tiburd As Samson has reviewed it, kindly merge this PR. Thanks

**Pull request merged**